.github/PULL_REQUEST_TEMPLATE/ttnn.md

```diff
@@ -0,0 +1,17 @@
+## Related Issues
+
+Please link related issues and describe the problem this pull-request is trying to resolve.
+
+## Description
+
+Please include a summary of the changes and the related issues. This template is for changes in the `ttnn` subdirectory.
+- Describe *what* changed in the pull-request
+- Describe *why* these changes were introduced
```

Comment:
@eyonland we can discuss in our team-sync whether this should really be contained within the issue itself. It sounds like maybe we should have a line here that describes *how* the changes in the pull-request solve the problem defined in the issue

Reply:
2c:
It should be within the issue so the one who works on the issue has The Context.
It should be within the PR so those reading are also in The Context.

Reply:
One format I've seen in the past is something like you see below.   This does restate the problem from the issue briefly, but also has the solution for what was done.

#Ticket Number: similar title to what is on the issue

Problem:
- Describe the problem
- Another bullet about the problem

Solution:
- What was changed
- another bullet about what was changed

Reply:
Thanks for the feedback, everyone. I've modified the template based on discussions we've had.